sdk/js-cloud-server/src/index.ts

```diff
@@ -33,8 +33,7 @@ export function initializeDevCycle(
 
     return new DevCycleCloudClient(sdkKey, options, {
         platform: options.platform || 'NodeJS',
```

Comment:
Do we want to remove this as a default from being nodejs? and make it something like "nativeJS"?

Reply:
you can take a look at the parent PR for that, but there is an interface for setting that directly.